extension/spark-doris-connector/src/main/scala/org/apache/doris/spark/sql/DorisOptions.scala

```diff
@@ -0,0 +1,11 @@
+package org.apache.doris.spark.sql
+
+object DorisOptions {
+  val beHostPort="beHostPort"
```

Comment:
The IP address of FE should be configured here, and the list of surviving BE nodes can be obtained through FE, and then communication training or other strategies during Stream load, connect to BE to execute load, and avoid the pressure of FE